man/genhtml.1

```diff
@@ -577,6 +579,13 @@ option
 .IR genhtml_missed .
 .RE
 
+.B \-\-dark\-mode
+.RS
+Use a light-display-on-dark-background colour scheme rather than teh default dark-display-on-light-background.
```

Comment:
Typo: teh => the, also please use the American English spelling "color" to be consistent with AE usage in rest of repository.